config/routes.rb

```diff
@@ -1,11 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  namespace :api do
-    namespace :v1 do
-      resources :games, only: [:index, :show, :create] do
-        resources :reviews, only: :index
-      end
+    namespace :api do
+      namespace :v1 do
+        resources :games, only: [:index, :create, :show] do
+          resources :reviews, only: [:new, :create]
```

Comment:
I notice that you're no longer allowing a `index` action for reviews. Please check that this change is intentional.